pkg/query/queryable.go

```diff
@@ -2,21 +2,22 @@ package query
 
 import (
 	"context"
+	querier2 "github.com/timescale/promscale/pkg/pgmodel/querier"
```

Comment:
Reason for name `querier2`?

Reply:
Sorry, that's by the editor. Will change these in the next commit.